person.rb

```diff
@@ -0,0 +1,30 @@
+# class for person
+class Person
+  attr_accessor :name, :age
+  attr_reader :id, :parent_permission
```

Comment:
Well done making the requested changes by the previous reviewer, However, I noticed there was an inadvertent addition by the previous reviewer.

[OPTIONAL] - This requirement is marked optional but highly recommended, according to the project requirement the only `attr_reader` we are meant to have is the `id`,  I would suggest that you remove the `:parent_permission` 🤓

Project requirement
![image](https://github.com/MargaretTechworld/OOP-School-Library/assets/78251927/2acce662-8a95-4920-8b26-ece4eac88acb)

Expected ✔️  (please correct that 💪)
```ruby
attr_accessor :name, :age
attr_reader :id
```